config/src/iroha.rs

```diff
@@ -20,8 +20,9 @@ view! {
         /// Private key of this peer
         #[view(ignore)]
         pub private_key: PrivateKey,
-        /// Disable coloring of the backtrace and error report on panic
-        pub disable_panic_terminal_colors: bool,
+        /// Disable coloring of the backtrace and error report on panic.
+        /// It is deprecated.
```

Comment:
I think this is better

```suggestion
        /// **Deprecated**.
        /// Disable coloring of the backtrace and error report on panic.
```